### ISSUE TITLE
Zen compact mode issues

### ISSUE THREAD

**Issue by NOCanoa** [zen-browser/desktop#68]
TLDR: full screen and compact mode have problems xD

When its on floating/windowed mode(compact mode), to trigger the side/top bar it is only possible at the top,
so its impossible to change tabs.

When in full screen compact mode:
to trigger the top/side bar it is possible at the side and the top, but its not possible to select any tabs because they disappear when the mouse leaves the top area.

when in full screen :
it is not possible to select any tab because they are not there (it is possible to show them when moving the mouse to the top off the screen but still not possible to change tabs)

**Comment by NOCanoa** [comment]
windowed:

https://github.com/user-attachments/assets/b7564b3f-a17b-4ff8-8014-670e17bb90f1

full full screen:

https://github.com/user-attachments/assets/f19d59ae-0f3c-435b-b217-667db9103192

**Comment by mauro-balades** [comment]
weird, it's working for me both fullscreen and windowed, maybe you have some userChrome.css that messes smth up? Could you try using it in a new profile? Ill try it again later on a windows enviroment

**Comment by NOCanoa** [comment]
nop is the exact same

**Comment by mauro-balades** [comment]
what version are you on?

**Comment by NOCanoa** [comment]
a5

**Comment by mauro-balades** [comment]
hmmm... alr ill look into it for the next release

**Comment by mauro-balades** [comment]
I think it should be fixed in https://github.com/zen-browser/desktop/commit/ccbb80ff4cbb3e8d678efa794e479b19e8d7b717. Ill reopen if the problem persists next release in a6.

**Comment by mauro-balades** [comment]
Now fixed (I hope)
